src/openrct2-ui/drawing/engines/opengl/TextureCache.h

```diff
@@ -217,6 +217,7 @@ class TextureCache final
     TextureCache();
     ~TextureCache();
     void InvalidateImage(uint32_t image);
+    // Note: for performance reasons, this returns a BasicTextureInfo over an AtlasTextureInfo (also to not expose the cache)
```

Comment:
The comment is a little misplaced, this interface doesn't have to provide justification to why it results this structure. It makes more sense to move this comment into the implementation.

Reply:
I pondered about that for a little and took the wrong choice hahah. Fixed it now, thanks.